src/components/pages/Places/bookmark/[bookmarkId]/BookmarkDetailList.tsx

```
@@ -73,37 +77,45 @@ const BookmarkDetailList = ({
+                  </DropdownMenuContent>
+                </DropdownMenu>
+              )
             }
             title={place.placeName}
             description={place.roadAddress}
+            LeftIcon={<TripAddButton place={place} />}
+            Pin={
+              mode !== 'schedule'
+                ? PinNumberIcon.bind(null, { number: idx + 1 })
+                : undefined
+            }
```

Biome (2.1.2)

[error] This JSX property is assigned multiple times.

This attribute is assigned again here.


(lint/suspicious/noDuplicateJsxProps)